bin/test-data/genesis/demo-tests/bitcoin-regtest/evm.json

```diff
@@ -16,10 +16,9 @@
   "coinbase": "0x0000000000000000000000000000000000000000",
   "starting_base_fee": 1000000000,
   "block_gas_limit": 30000000,
-  "genesis_timestamp": 1705957200,
   "block_timestamp_delta": 2,
```

Comment:
No need for `block_timestamp_delta` anymore

Reply:
yes